test/cpp/end2end/cfstream_test.cc

```diff
@@ -261,6 +294,119 @@ TEST_F(CFStreamTest, NetworkTransition) {
   sender.join();
 }
 
+// Network flaps while RPCs are in flight
+TEST_F(CFStreamTest, NetworkFlapRpcsInFlight) {
```

Comment:
Suggest getting a C++ team's review.

Can this run on CI?

Reply:
fixed.